generator/src/main.rs

```diff
@@ -1601,10 +1621,14 @@ impl Parser {
             out.has_pointer |= member.ptr_depth != 0 || self.handles.contains(&member.ty);
             out.has_graphics |= member.ty == "XrSession" || member.ty == "XrSwapchain";
             out.has_array |= member.static_array_len.is_some();
+            out.has_non_default |= member.ty == "XrTime" || member.ty == "XrDuration";
```

Comment:
Should we impl `Default` for these instead? At least for `Duration` it seems totally reasonable. Maybe not `Time`, per std's precedent on `Instant`.